packages/inference/src/tasks/cv/imageToVideo.ts

```diff
@@ -0,0 +1,24 @@
+import type { ImageToVideoInput } from "@huggingface/tasks";
+import { resolveProvider } from "../../lib/getInferenceProviderMapping.js";
+import { getProviderHelper } from "../../lib/getProviderHelper.js";
+import type { BaseArgs, Options } from "../../types.js";
+import { innerRequest } from "../../utils/request.js";
+import { makeRequestOptions } from "../../lib/makeRequestOptions.js";
+
+export type ImageToVideoArgs = BaseArgs & ImageToVideoInput;
+
+/**
+ * This task reads some text input and outputs an image.
+ * Recommended model: lllyasviel/sd-controlnet-depth
```

Comment:
Nit, 
```suggestion
 * Recommended model: Wan-AI/Wan2.1-I2V-14B-720P
```
either change to a image-to-video model or just remove the comment